sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -834,6 +834,14 @@ class Analyzer(
           v2Catalog.asTableCatalog, ident,
           UnresolvedRelation(alter.tableName),
           Seq(TableChange.setProperty("location", newLoc)))
+
+      case describe @ DescribeTableStatement(
```

Comment:
Any reason this is added inside `ResolveAlterTable`?

Reply:
Good call - moved to a separate rule.

Reply:
(think this thread is attached to a commit. I'm going to resolve the conversation - please see the most up to date diff to verify the change went through)